hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -53,6 +54,13 @@
 INDEXER_FUNCTION_NAME = os.environ["INDEXER_FUNCTION_NAME"]
 WRITEBACK_QUEUE_URL = os.environ["WRITEBACKS_QUEUE_URL"]
 
+# Initialize hmaconfig at module level. Mounted SubApps need not initialize
+# their own HMAConfigs.
+HMAConfig.initialize(HMA_CONFIG_TABLE)
+
+functionality_mapping = get_pytx_functionality_mapping()
```

Comment:
(dumb question): This might be a significant amount of latency since it's making dynamodb calls. Are we already doing this kind of work at the outer level?

Reply:
Since this is done before the lambda handler is called, the latency is ammortized over the lifetime of a lambda instance. During such a lifetime, it probably handles thousands (if not tens of thousands) of requests. 

That, and DynamoDB is quite performant. Individual gets are p99 10ms. I don't think scans for a single partition will be much larger. Config table is expected to be tiny.